File: jar/src/main/kotlin/jline/solvers/des/DES.java
/*
 * Copyright (c) 2012-2026, QORE Lab, Imperial College London
 * All rights reserved.
 */

package jline.solvers.des;

import jline.lang.Network;
import jline.solvers.SolverOptions;

/**
 * DES is an alias for SolverDES (Discrete Event Simulation solver).
 */
public class DES extends SolverDES {

    public DES(Network model, String method) {
        super(model, method);
    }

    public DES(Network model, SolverOptions options) {
        super(model, options);
    }

    public DES(Network model, Object... varargin) {
        super(model, varargin);
    }

    public DES(Network model) {
        super(model);
    }
}

File: matlab/src/solvers/DES.m
classdef DES < SolverDES
    % DES - Alias for SolverDES (Discrete Event Simulation solver)
    %
    % Copyright (c) 2012-2026, Imperial College London
    % All rights reserved.

    methods
        function self = DES(model, varargin)
            % DES(MODEL, VARARGIN)
            self@SolverDES(model, varargin{:});
        end
    end
end
